README.md

```diff
@@ -136,11 +136,36 @@ The unique characteristics of cases of this class are as follows:
 |Small cell cancer of the lung, somatic, 182280 (3)|RB1|614041|13q14.2|
 |Small-cell cancer of lung (2)|SCLC1|182280|3p23-p21|
 
-
 **All known cases**:
 There is a spreadsheet which collates all known cases as of 2024/11/18: [google sheet](
 https://docs.google.com/spreadsheets/d/1hKSp2dyKye6y_20NK2HwLsaKNzWfGCMJMP52lKrkHtU/). The MIMs of the known cases are: `159595`, `182280`, `607107`, and `615830`.
 
+**Additional notes**:
+Note that unlike the other `review.tsv` cases, for this one, a single case spans multiple rows. The cases can be 
+identified easily because they are enumerated with a leading number in the `value` column, e.g. "1: " for the first case
+, and so on.
+
+Also, see note in section "3. D2G: somatic" about intersection between these two cases. 
+
+#### 3. D2G: somatic
+Happens when all conditions were met for this association to be considered disease-defining, but the word 'somatic' is 
+in the phenotype label. These are somatic mutation entries in OMIM, but typically they are germline mutations, and that 
```

Comment:
this sounds confusing `These are somatic mutation entries in OMIM, but typically they are germline mutations,...`.

Reply:
Yeah, I agree. Do you think you or maybe Sabrina could describe better? Or maybe I understand correctly and this updated description is OK?:

> #### 3. D2G: somatic
> Happens when all conditions were met for this association to be considered disease-defining, but the mutation is a somatic cell mutation, rather than a germline mutation. This is indicated by the appearance of the word 'somatic' in the label of the phenotype MIM in the association. These cases should be looked at because if they are a logical contradiction, as we use the predicate "causal germline mutation in" to describe the association, but these are not germline mutations.
>

Reply:
Sure, minor modification to your last post:
```
Happens when all conditions were met for this association to be considered disease-defining, but the mutation is a somatic cell mutation, rather than a germline mutation. This is indicated by the appearance of the word 'somatic' in the label of the phenotype MIM in the association. These cases should be reviewed because currently any association meeting the criteria to be considered disease-defining is also considered a germline mutation and the association is represented in the omim.owl file using the property 'is causal germline mutation in' (RO:0004013).
```

Reply:
Done!